### ISSUE TITLE
librepo fails to build against new attr 2.4.48

### ISSUE THREAD

**Issue by david-geiger** [rpm-software-management/librepo#121]
Hi,

Since latest attr package version 2.4.48 librepo fails to build due to a missing include:

`/home/pterjan/rpmbuild/BUILD/librepo-librepo-1.7.20/librepo/downloader.c:33:10: fatal error: attr/xattr.h: No such file or directory
 #include <attr/xattr.h>`

**Comment by ignatenkobrain** [comment]
are you use that you have libattr-devel installed?

**Comment by ignatenkobrain** [comment]
http://git.savannah.nongnu.org/cgit/attr.git/commit/?id=7921157890d07858d092f4003ca4c6bae9fd2c38

yeah, seems they removed that

**Comment by david-geiger** [comment]
Yes libattr-devel is installed but they have removed attr/xattr.h and now we should use sys/xattr.h from glibc

**Comment by ignatenkobrain** [comment]
@david-geiger could you try #122?

**Comment by david-geiger** [comment]
Thanks! now it build fine!

[librepo_build_log.txt](https://github.com/rpm-software-management/librepo/files/1713660/librepo_build_log.txt)
